avocado/utils/disk.py

```diff
@@ -118,10 +118,18 @@ def get_disks():
     :returns: a list of paths to the physical disks on the system
     :rtype: list of str
     """
-    json_result = process.run("lsblk --json --paths --inverse")
+    json_result = process.run('lsblk --json --paths --inverse')
     json_data = json.loads(json_result.stdout_text)
-    return [str(disk["name"]) for disk in json_data["blockdevices"]]
-
+    # Create an empty list to store the disk names
```

Comment:
please remove this comment and elsewhere as well

Reply:
Done